submission/submitter/src/test/kotlin/ac/uk/ebi/biostd/submission/service/AccNoServiceTest.kt

```diff
@@ -29,25 +29,31 @@ private const val PROJECT_PATTERN = "!{ABC-}"
 @ExtendWith(MockKExtension::class)
 class AccNoServiceTest(
     @MockK private val service: PersistenceService,
-    @MockK private val privilegesService: IUserPrivilegesService
+    @MockK private val privilegesService: IUserPrivilegesService,
 ) {
     private val accNoPatternUtil: AccNoPatternUtil = AccNoPatternUtil()
-    private val testInstance = AccNoService(service, accNoPatternUtil, privilegesService)
+    private val testInstance = AccNoService(service, accNoPatternUtil, privilegesService, subBasePath = null)
 
     @AfterEach
     fun afterEach() = clearAllMocks()
 
-    @ParameterizedTest(name = "prefix is {0} and numeric value is {1}")
-    @CsvSource(
-        "S-DIXA-AN-002, S-DIXA-AN-/002/S-DIXA-AN-002",
-        "S-BSST11, S-BSST/011/S-BSST11",
-        "S-DIXA-011, S-DIXA-/011/S-DIXA-011",
-        "1-AAA, 1-AAA/000/1-AAA",
-        "S-SCDT-EMBOJ-2019-103549, S-SCDT-EMBOJ-2019-/549/S-SCDT-EMBOJ-2019-103549",
-        "S-SCDT-EMBOR-2017-44445V1, S-SCDT-EMBOR-2017-44445V/001/S-SCDT-EMBOR-2017-44445V1"
-    )
-    fun getRelPath(value: String, expected: String) {
-        assertThat(testInstance.getRelPath(accNoPatternUtil.toAccNumber(value))).isEqualTo(expected)
+    inner class GetRelPath {
+        @ParameterizedTest(name = "prefix is {0}, numeric value is {1} and submission base path is {2}")
```

Comment:
I noticed this isn't quite accurate since {0} is actually the "numeric value" which is actually the accession 😅. Also, {1} is the expected relPath. I think we should change it to:

"{0} = accession, {1} = base path, {2}=expected rel path"

Which is shorter, clearer and follows the parameters -> expected result pattern. WDYT?